src/wp-includes/theme.php

```diff
@@ -4416,3 +4445,45 @@ static function ( $active, WP_Customize_Panel $panel ) {
 		2
 	);
 }
+
+/**
+ * Loads view transitions for the current theme, if configured.
+ *
+ * @since 6.9.0
+ */
+function wp_load_view_transitions() {
+	if ( ! current_theme_supports( 'view-transitions' ) ) {
+		return;
+	}
+
+	$stylesheet = file_get_contents( ABSPATH . WPINC . '/view-transitions.css' );
+
+	// Use an inline style to avoid an extra request, also because it's just 1 line of CSS.
```

Comment:
+1 to inline, its more than 1 line of CSS to maybe leave that part off
```suggestion
	// Use an inline style to avoid an extra request.
```